front/temporal/labs/activities.ts

```diff
@@ -278,21 +318,15 @@ export async function processGoogleDriveTranscriptActivity(
       return innerArray.find((item) => item.type === "agent_message");
     }
   );
-  const fullAnswer = agentMessage ? agentMessage[0].content : "";
-
-  const hasExistingHistory =
-    await transcriptsConfiguration.fetchHistoryForFileId(fileId);
-  if (hasExistingHistory) {
-    localLogger.info(
-      "[processGoogleDriveTranscriptActivity] History record already exists. Stopping."
-    );
-    return;
-  }
+  const markDownAnswer =
+    agentMessage && agentMessage[0].content ? agentMessage[0].content : "";
+  const htmlAnswer = marked.parse(markDownAnswer);
```

Comment:
You can't make that assumption

Reply:
As per IRL actually we can make the reasonable assumption that assistant outputs is markdown-esque. So likely good :+1:

Reply:
After a few trials, the worst case scenario (tried to make it answer YML, HTML, etc) is that the output is stripped of tags because of HTML sanitation.

I'd recommend to leave it like this with the hypothesis that regular usage will be 'send me a nice looking email with my processed transcript'. That's a decision of usability vs customizability, let's go for the most usable one for now.